src/profile/utils.js

```diff
@@ -0,0 +1,266 @@
+import { API_BASE_URL, YEARS_OF_EXPERIENCE } from './constants.js';
+
+//function to get the profile differences
+async function getProfileDiffs() {
+  try {
+    const profileDiffsResponse = await fetch(`${API_BASE_URL}/profileDiffs`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+    const { profileDiffs } = await profileDiffsResponse.json();
+    return { profileDiffs };
+  } catch (error) {
+    alert(`Error: ${error}`);
+  }
+}
+
+//resusable function to extract the key property field and displaying the data
+function formatPropertyField(property) {
+  return property
+    .split('_')
+    .map((x) => x[0].toUpperCase() + x.slice(1))
+    .join(' ');
+}
+
+//to get the item from the data
+function getDataItem(data, itemName) {
+  const item = data[itemName];
+
+  if (item) {
+    return item;
+  } else {
+    if (itemName === YEARS_OF_EXPERIENCE && item === 0) return item;
+    else return '';
+  }
+}
+
+//To display the list of data(old and new)
+function displayList(data, userInfoList) {
+  for (const listItem in data) {
+    const li = document.createElement('li');
```

Comment:
Move createElement as a reusable method. This method will accept the HTML element name, content, attributes such as classname